test-core/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
-cmake_minimum_required (VERSION 3.1)
 project (CoreCVSTests)
 
+set(MODULE_NAME corecvs)
+
+
 MESSAGE( STATUS "Including GTest on Tests build" )
 find_package( GTest REQUIRED)
 include_directories (${GTest_INCLUDE_DIR}/..)
```

Comment:
It's not needed as the gtest_main below is a target, and hence it activates the proper include path for the module "${MODULE_NAME}_tests" and linking proper library as well.